uberfire-extensions/uberfire-layout-editor/uberfire-layout-editor-client/src/main/java/org/uberfire/ext/layout/editor/client/infra/LayoutEditorFocusController.java

```diff
@@ -0,0 +1,92 @@
+package org.uberfire.ext.layout.editor.client.infra;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import org.uberfire.ext.layout.editor.client.components.container.Container;
+
+import com.google.gwt.core.client.Scheduler;
+import com.google.gwt.dom.client.Style;
+import com.google.gwt.dom.client.Style.Overflow;
+
+import elemental2.dom.HTMLElement;
+import jsinterop.base.Js;
+
+@ApplicationScoped
```

Comment:
Could you move this above the class name?

Reply:
oops, done, thanks @paulovmr